unittests/autoconfig.py

```diff
@@ -56,6 +56,11 @@ class cxx_parsers_cfg(object):
         if 'msvc9' == gccxml.compiler:
             gccxml.define_symbols.append('_HAS_TR1=0')
 
+if cxx_parsers_cfg.gccxml.xml_generator:
```

Comment:
Ok I get it.

For the moment, gccxml is still the "default" xml generator. I was rather conservative here, because I wanted people to have time to update their code. I plan to make castxml the default xml generator for release 1.8.0 (and keep gccxml as the second option).

```pygccxml.utils.find_xml_generator()``` makes a wild guess and tries to find a xml generator. If you have gccxml installed it ill pick that up first (by default).
Then, you are overriding the values with the config (which is the recommended way to do it, I just added ```find_xml_generator()``` to help beginners).

So your change is legitimate. Perfect.